src/sentry/static/sentry/app/components/onboardingWizard/taskConfig.tsx

```diff
@@ -95,11 +95,24 @@ export function getOnboardingTasks(
     {
       task: OnboardingTaskKey.SECOND_PLATFORM,
       title: t('Add a second platform'),
-      description: t('Add Sentry to a second platform'),
-      detailedDescription: t('Capture errors from both your front and back ends.'),
+      description: t(
+        `Add a new project to capture errors from both your front and back ends for your other platforms`
+      ),
       skippable: true,
       requisites: [OnboardingTaskKey.FIRST_PROJECT, OnboardingTaskKey.FIRST_EVENT],
       actionType: 'app',
+      getPendingLocation: (org, project) => {
+        // It's confusing to continue linking to the "create new project"
+        // page after the user has created the new project. Instead, link to the install
+        // instructions for the corresponding project to prompt the user to send an event
+        // to the new project.
+        const projectSlug =
+          (project && org?.projects?.find(p => p.id === `${project}`)?.slug) ??
+          ':projectId';
```

Comment:
Now we check for the presence of each attribute instead of inappropriately using the typescript non-null assertion operator, which caused the original issue: https://sentry.io/organizations/sentry/issues/1724187385/?project=11276&referrer=slack

I'm not sure why this issue didn't surface when I originally tested these changes locally. But when working on a fix for the not-actually-fixing fix, I was able to reproduce this issue, and adding the optional chaining to `org.project?`, as you see here, resolved the issue locally. (I'm still going to test in staging before I feel comfortable rolling this out to prod. But what I still don't understand is, why is the optional chaining necessary here, when projects attribute is *not* optional on the Organization type? Wouldn't that have caused a type error when the organization object was first initialized, rather than when the missing projects attribute was referenced?

Reply:
Sorry I led you astray here on the non-fix-fix. Our types for `Organization` may not be 100% accurate because for views that use the `LightweightOrganization`, `projects` is async.